assets/section-footer.css

```diff
@@ -463,16 +464,54 @@ noscript .localization-selector.link {
   }
 }
 
-.footer-block-image > img {
+.brand-information__image-wrapper {
+  box-shadow: var(--media-shadow-horizontal-offset)
+    var(--media-shadow-vertical-offset)
+    var(--media-shadow-blur-radius)
+    rgba(var(--color-shadow), var(--media-shadow-opacity));
+  margin-bottom: calc(2rem + var(--media-shadow-vertical-offset) * var(--media-shadow-visible));
+  overflow: hidden !important;
+}
+
+.brand-information__image-wrapper img {
+  display: block;
+  height: auto;
+  max-width: 100%;
+}
+
+.footer-block-brand-information {
+  text-align: left;
+}
+
+.footer-block:only-child .footer-block-brand-information {
+  text-align: center;
+}
+
+.footer-block:only-child .brand-information__image-wrapper {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.footer-block-image > img,
+.footer-block-brand-information > img {
   height: auto;
 }
 
+.footer-block:only-child .footer-block-brand-information .footer__list-social.list-social {
+  justify-content: center;
```

Comment:
I think here we would need to also remove the negative margin left to make sure it's properly centered. Or add a negative `margin-right`
[video](https://screenshot.click/09-30-64en6-l8g6r.mp4)

Reply:
Updated with `margin-right`. Removing `margin-left` would break the layout when icons are left aligned :D